src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+/**
+ * Class that represents exceptions specific to Duke.
+ */
+public class DukeException extends Exception {
+    /**
+     * Constructor for a DukeException.
+     *
```

Comment:
I think there should be a space between class definition and the function header